server/routerlicious/packages/services-utils/src/redisClientConnectionManager.ts

```diff
@@ -32,7 +32,7 @@ export class RedisClientConnectionManager implements IRedisClientConnectionManag
 	};
 
 	constructor(
-		redisOptions?: Redis.RedisOptions,
+		redisOptions?: Redis.RedisOptions & Redis.ClusterOptions,
 		redisConfig?: any,
```

Comment:
The strategy would still be applied I think, because it checks if this.redisOptions.retryStrategy is defined AFTER the if/elif/else that checks whether redisConfig and/or redisOptions are defined. Then after that, this.redisOptions.clusterRetryStrategy is set to the same value as retryStrategy, so it should always work.
The reason I didn't modify the constructor is because we only need to set clusterRetryStrategy when clustering is enabled